hack/make-release-notes.sh

```diff
@@ -38,6 +42,10 @@ echo "Installation"
 echo "------------"
 echo "To install this release, refer to the instructions at ${readme}."
 echo
+echo "Merged pull requests"
```

Comment:
I think this is definitely lower priority than contributors list + release asset links.

Maybe move this to the end?

And maybe if possible we should make it collapsible, like:

<details>
<summary>Pull Requests</summary>
<pre>
Html list here
</pre>
</details>

with

```
<details>
<summary>...</summary>
<pre>
...
</pre>
</details>
```

Reply:
Yes, PRs have low priority, so I moved it to the end. I think it even works with markdown in the collapsed region, if a blank line is inserted. I haven't checked how the generated output renders on github though...